src/react-components/room/RoomSidebar.js

```diff
@@ -135,7 +136,65 @@ export function SceneInfo({ accountId, scene, showAttributions, canChangeScene,
   );
 }
 
-export function RoomSidebar({ room, accountId, onClose, canEdit, onEdit, onChangeScene }) {
+function DebugSceneDropzone({ hubChannel }) {
+  const [lastBlobUrl, setLastBlobUrl] = useState();
+  const [dropping, setDropping] = useState(false);
+
+  const handleDragEnter = e => {
+    e.preventDefault();
+    setDropping(true);
+  };
+  const handleDragLeave = e => {
+    e.preventDefault();
+    setDropping(false);
+  };
+
+  const handleDragOver = e => {
+    e.preventDefault();
+    e.dataTransfer.dropEffect = "move";
+  };
+
+  const handleDrop = e => {
+    e.preventDefault();
+    URL.revokeObjectURL(lastBlobUrl);
+    const url = URL.createObjectURL(e.dataTransfer.files[0]);
+    hubChannel.updateScene(url);
```

Comment:
Yep, it does end up updating things on the server. Working around that is pretty hairy so I decided it was fine for a debug flag. It would break on refreshing the page, but I added a check to only load blob files once you are in the scene. So when you refresh a room previously in this mode you will get the blank loading scene instead.